tests/src/test/scala/scalaz/std/OptionTest.scala

```diff
@@ -25,6 +25,7 @@ object OptionTest extends SpecLite {
   checkAll("Option", cobind.laws[Option])
   checkAll("Option", align.laws[Option])
   checkAll("Option", band.laws[Option[ISet[Int]]])
+  checkAll("Option", semilattice.laws[Option[ISet[Int]]])
```

Comment:
We can remove `checkAll("Option", band.laws[Option[ISet[Int]]])` if `trait Semilattice extends Band` and `semilattice laws` include `band laws`